samples/test-app-screenshot-test/src/androidTest/kotlin/com/avito/android/runner/ScreenshotTestRunner.kt

```diff
@@ -0,0 +1,41 @@
+package com.avito.android.runner
+
+import android.content.Intent
+import android.os.Bundle
+import com.avito.android.runner.annotation.resolver.AnnotationResolversBasedMetadataInjector
+import com.avito.android.runner.annotation.resolver.NetworkingResolver
+import com.avito.android.runner.annotation.resolver.TestMetadataAnnotationResolver
+import com.avito.android.runner.annotation.resolver.TestMetadataInjector
+import com.avito.android.screenshot_test.internal.ScreenshotRunner
+import com.avito.android.test.report.BundleArgsProvider
+
+class ScreenshotTestRunner : InHouseInstrumentationTestRunner() {
+
+    override fun createRunnerEnvironment(arguments: Bundle): TestRunEnvironment {
+        return provideEnvironment(
+            argumentsProvider = BundleArgsProvider(bundle = arguments)
+        )
+    }
+
+    override val metadataToBundleInjector: TestMetadataInjector = AnnotationResolversBasedMetadataInjector(
+        setOf(
+            TestMetadataAnnotationResolver(),
+            NetworkingResolver()
```

Comment:
Do we mock network in screenshot tests?

Reply:
No, just was not sure if screenshot tests can work without it.

Reply:
It's used for network test rules: https://avito-tech.github.io/avito-android/docs/test_framework/mocking/#networking
RuleResolver parses test, looking for annotations and adds information to test runner arguments.
A test runner reads it and mocks a network. As I see it's not relevant to `ScreenshotTestRunner`

Reply:
If I remove NetworkingResolved there are "no tests were found". After returning - it works again.

Reply:
@eugene-krivobokov `networkType` is a mandatory argument now